submissions/lalalilulu/dom-api-menu/script.js

```diff
@@ -0,0 +1,227 @@
+        description: "Краска для волос",
+        price: 249,
+        image: "./images/coloration/photo_1.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "Schwarzkopf Got2b Bright/Pastel Набор для тонирования волос",
+        description: "Набор для тонирования волос",
+        price: 299,
+        image: "./images/coloration/photo_2.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Magic Retouch",
+        description: "Краска для волос",
+        price: 204,
+        image: "./images/coloration/photo_3.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Colorista Spray",
+        description: "Тонирующий спрей для волос",
+        price: 299,
+        image: "./images/coloration/photo_4.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Crystal Clear",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_1.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pretzel Brown",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_2.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Wrapstar Hair-Rings Snake It Off",
+        description: "Резинка с лентой",
+        price: 1092,
+        image: "./images/accessories/photo_3.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pinkerbell",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_4.jpg"
+    }];
+
+function createElementWithClass(tagName, cssClass) {
+    const element = document.createElement(tagName);
+    element.classList.add(cssClass);
+    return element;
+}
+
+function createContentItem(product) {
+    const productElement = createElementWithClass("div", "content-item");
+
+    const itemImg = createElementWithClass("img", "content-item-image");
+    itemImg.setAttribute("src", product.image);
+    itemImg.setAttribute("alt", product.title);
+
+    const itemTitle = createElementWithClass("a", "content-item-title");
+    itemTitle.setAttribute("href", "#");
+    itemTitle.textContent = product.title;
+
+    const itemDescription = createElementWithClass("div", "content-item-description");
+    itemDescription.textContent = product.description;
+
+    const itemPrice = createElementWithClass("div", "content-item-price");
+    itemPrice.textContent = product.price;
+    const rubleSign = createElementWithClass("span", "rur");
+    rubleSign.textContent = "p";
+    itemPrice.appendChild(rubleSign);
+
+    productElement.append(itemImg, itemTitle, itemDescription, itemPrice);
+    return productElement;
+}
+
+    const content = document.getElementById("content");
+
+    document.querySelector(".menu-list").addEventListener("click", function ({target}) {
+    const productType = target.textContent;
```

Comment:
it's better to move these all operations into one function. It would be easier to test these all stuff. You can't do it now. It's just a recommendation to the next your tasks